tests/e2e/tests/test_sanity_portforward.py

```diff
@@ -155,3 +205,80 @@ def test_katib_experiment(self, cluster, region):
             raise AssertionError("Expected K8sApiException Not Found")
         except K8sApiException as e:
             assert "Not Found" == e.reason
+
+    @pytest.mark.parametrize(
+        "framework_name, image_name, ipynb_notebook_file, expected_output", testdata
+    )
+    def test_ack_crds(
+        self,
+        region,
+        metadata,
+        notebook_server,
+        framework_name,
+        image_name,
+        ipynb_notebook_file,
+        expected_output,
+    ):
+        """
+        Spins up a DLC Notebook and checks that the basic ACK CRD is installed. 
+        """
+        nb_list = subprocess.check_output(
+            f"kubectl get notebooks -n {DEFAULT_USER_NAMESPACE}".split()
+        ).decode()
+
+        metadata_key = f"{framework_name}-notebook_server"
+        notebook_name = notebook_server["NOTEBOOK_NAME"]
+        assert notebook_name is not None
+        assert notebook_name in nb_list
+        print(notebook_name)
+
+        sub_cmd = f"jupyter nbconvert --to notebook --execute ../uploaded/{ipynb_notebook_file} --stdout"
+        cmd = f"kubectl -n kubeflow-user-example-com exec -it {notebook_name}-0 -- /bin/bash -c".split()
+        cmd.append(sub_cmd)
+
+        output = subprocess.check_output(cmd, stderr=subprocess.STDOUT).decode()
+        print(output)
+        # The second condition is now required in case the kfp test runs before this one.
```

Comment:
unless I missed something, there is no dependency between kfp test and ack test so this can happen?

Reply:
As in they could run out of order ? The current assert is safe and should be able to handle any scenario

Reply:
got it, where is expected_output defined?

Reply:
in the testdata line 64 above. The format was very useful for the notebook_images test so I kept it, maybe not so necessary in this test.